backends-velox/src/main/scala/io/glutenproject/execution/ShuffledHashJoinExecTransformer.scala

```diff
@@ -152,6 +157,20 @@ case class ShuffledHashJoinExecTransformer(
     copy(left = newLeft, right = newRight)
 }
 
+case class VeloxBroadcastBuildSideRDD(
```

Comment:
This case class is not specific to hash join and can be moved to generic place or to a new file.

Reply:
I'm moving it out from this file. If you think it can be reused for BNLJ.